src/stactools/sentinel2/stac.py

```diff
@@ -128,7 +129,8 @@ def create_item(
         metadata = metadata_from_granule_metadata(
             granule_href, read_href_modifier, tolerance
         )
-    created = datetime.now().strftime("%Y-%m-%dT%H:%MZ")
```

Comment:
🙃

Reply:
seeing that used so frequently was the entire reason I created that pystac.utils method